.github/workflows/release.yml

```
@@ -0,0 +1,48 @@
+name: goreleaser
```

Is it workflow for release?
```suggestion
name: "Release scylla-bench"
```